app/javascript/packs/application.js

```diff
@@ -7,7 +7,9 @@ require("@rails/ujs").start()
 require("turbolinks").start()
 require("@rails/activestorage").start()
 require("channels")
-
+require('jquery')
```

Comment:
```suggestion
require("jquery")
```
If you started using double quotes then be consistent